docs/appendix/release-notes.md

```diff
@@ -2,7 +2,7 @@
 title: "Release Notes"
```

Comment:
delete this file

Reply:
Andrew and I discussed leaving it as draft for now as we compare options for automation etc. going forward

Reply:
Ok to delete also, can always consult git history down the line if needed.

Reply:
done